portfolio/src/main/java/com/google/sps/servlets/CheckStatus.java

```diff
@@ -21,8 +23,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
         out.println(new Gson().toJson(true));
+<<<<<<< HEAD
+<<<<<<< HEAD
+    }
+    else {
+=======
     }else{
+>>>>>>> 1e9b006429eac588504484c5c9e9fe1332bdf466
+=======
+    }
+    else {
+>>>>>>> 59e25543aa73e9ebc121f9f644e29d02bc0f27ae
         out.println(new Gson().toJson(false));
     }
   }
+}
+        out.println(new Gson().toJson("Logged in as " + userService.getCurrentUser().getEmail() + "."));
+    }
+else {
+        out.println(new Gson().toJson("Currently logged in as Guest."));
+    }
+  }
```

Comment:
:/ The indentations looks weird here. Is this meant to be here in the first place? I thought we were only returning true or false from this endpoinnt.